### PR TITLE
Final params for BqETHSubscription deployment on mainnet

### PR DESCRIPTION
**Pull request by cygnusv**
**Type of PR:**
- [ ] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Other

**Required reviews:** 
- [ ] 1
- [ ] 2
- [X] 3

**What this does:**
> High-level idea of the changes introduced in this PR. 
> List relevant API changes (if any), as well as related PRs and issues.

**Issues fixed/closed:**
> - Fixes #...

**Why it's needed:**
> Explain how this PR fits in the greater context of the NuCypher Network.
> E.g., if this PR address a `nucypher/productdev` issue, let reviewers know!

**Notes for reviewers:**
> What should reviewers focus on? 
> Is there a particular commit/function/section of your PR that requires more attention from reviewers?

### CODE REVIEW

**Review by manumonti — APPROVED** [review]

**Comment by cygnusv** [comment]
This is currently deployed on Mainnet

**Comment by derekpierre** [comment]
> This is currently deployed on Mainnet

Separate from this PR:
1. We've been using testnet tags for tracking the code currently deployed on the various testnets:
- https://github.com/nucypher/nucypher-contracts/tree/lynx
- https://github.com/nucypher/nucypher-contracts/tree/tapir
- https://github.com/nucypher/nucypher-contracts/tree/mainnet

Not sure if we updated `lynx` and `mainnet` tags, but those should be updated accordingly. 

2. It's likely that `tapir` contracts are now older than `mainnet`, so `tapir` should be updated.

File: deployment/constructor_params/mainnet/bqeth.yml [hunk 1]
```diff
@@ -23,15 +25,17 @@ constants:
...
+
   # - Fee parameters:
   INITIAL_BASE_FEE_RATE: 4050925925925  # $0.35 per day, in DAI units per second (in Python: 35*10**16 // 86400)
-  ENCRYPTOR_FEE_RATE: 63419583967  # $2 per year, in DAI units per second (in Python: 2 * 10**18 // 86400 // 365)  
+  ENCRYPTOR_FEE_RATE: 63419583967  # $2 per year, in DAI units per second (in Python: 2 * 10**18 // 86400 // 365)
   BASE_FEE_RATE_INCREASE_PER_PERIOD: 247  # 5%/year ~ 2.47%/semester, expressed in basis points (0.01%)
   # - Duration parameters --> 1 period = 6 months
-  SIX_MONTHS_IN_SECONDS: 15552000  # 180 days (~6 months)
+  SIX_MONTHS_IN_SECONDS: 15811200  # 183 days (~6 months)
   THREE_MONTHS_IN_SECONDS: 7776000  # 90 day (~3 months)
```
Comment (derekpierre) [inline deployment/constructor_params/mainnet/bqeth.yml:39]: The 3-month value was relateive to the 6-month value, did you want to update that as well?

File: deployment/constructor_params/mainnet/free-fee-model.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+deployment:
+  name: mainnet-free-fee-model
+  chain_id: 137
+
+artifacts:
+  dir: ./deployment/artifacts/
+  filename: free-fee-model.json
```
Comment (derekpierre) [inline deployment/constructor_params/mainnet/free-fee-model.yml:7]: With this deployment, was the `free-fee-model.json` file committed to the repos? I don't see it in this PR - but perhaps in a different PR?

Comment (derekpierre) [reply deployment/constructor_params/mainnet/free-fee-model.yml:7]: Oh it is in `mainnet.json`.

**Pull request merged**